include/conf/load.hpp

```diff
@@ -41,6 +41,20 @@ namespace color_coded
 
         return line;
       }
+
+      void make_absolute(std::string &line, std::string &next, const fs::path &base) {
+        static const std::regex reg_dual {
+          "\\s*(-I|-isystem|-iquote)\\s*$",
+            std::regex::optimize
+        };
+
+        std::smatch match;
+        if (std::regex_search(line, match, reg_dual)) {
+          next = fs::absolute(next.c_str(), base).string();
+        } else {
+          line = make_absolute(line, base);
+        }
+      }
     }
```

Comment:
Will you please update the braces and `const` positioning to match the rest of the file? We use Allman-like braces, with a one-line exception, and `const` always goes to the right. You can find examples of each of these within this file.